app/domain/operations/premium_credits/find_aptc.rb

```diff
@@ -161,7 +161,7 @@ def utilized_aptc(aptc_grant)
         coinciding_enrollments.reduce(0.0) do |sum, previous_enrollment|
           th_enrollment = TaxHouseholdEnrollment.where(enrollment_id: previous_enrollment.id, tax_household_id: aptc_grant.tax_household_id).first
           next sum if th_enrollment.blank?
-          value = round_down_float_two_decimals(th_enrollment.available_max_aptc)
+          value = round_down_float_two_decimals(th_enrollment.available_max_aptc || 0.0)
```

Comment:
@jayreddy519 where are we setting `th_enrollment.available_max_aptc`? Would it be better to default to zero?

Reply:
yeah @jacobkagon  we can change that, 

0/0.0 will return 0.0 on line 167
`sum += (value > 0.0 ? value : 0.0)`